docs/src/components/Layout/Header.tsx

```diff
@@ -36,22 +32,34 @@ export const Header = ({
   }, [showSearch]);
 
   return (
-    <Flex as="header" className="docs-header">
-      <MenuButton expanded={expanded} setExpanded={setExpanded} />
+    <Flex
+      as="header"
+      className={`docs-header ${
+        alwaysCollapsible ? 'always-collapsible' : null
+      }`}
+    >
+      <MenuButton
+        expanded={expanded}
+        setExpanded={setExpanded}
+        alwaysCollapsible={alwaysCollapsible}
+      />
 
       <Sidebar
         expanded={expanded}
         setExpanded={setExpanded}
+        alwaysCollapsible={alwaysCollapsible}
         platform={platform}
       />
 
-      <LogoLink platform={platform} />
+      <LogoLink platform={platform} alwaysCollapsible={alwaysCollapsible} />
```

Comment:
Instead of passing `alwaysCollapsible` around, we should just add a classname at the nearest ancestor DOM node and style it from there.

Reply:
Let's chat about this one, having it show up once the root of the menu component would be ideal but hesitated on how to make the necessary scss updates